core/src/commonTest/kotlin/com/sourcepoint/mobile_core/CoordinatorTest.kt

```diff
@@ -573,18 +573,34 @@ class CoordinatorTest {
 
     @Test
     fun propertyWithPreferencesCampaign() = runTestWithRetries {
-        return@runTestWithRetries //TODO: this test doesn`t work on prod
         val coordinator = getCoordinator(
             accountId = 22,
             propertyId = 38984,
             propertyName = "dmytro.tests.mobile.preferences",
             campaigns = SPCampaigns(preferences = SPCampaign()),
             spClient = SourcepointClient(
                 accountId = 22,
-                propertyId = 38984
+                propertyId = 38984,
+                PlatformHttpClient.create().engine
```

Comment:
@Nevazhnovu is this line necessary? The other tests don't seem to have it.

Reply:
@andresilveirah Yes, it's essential; iOS test fails without it